web/spec/storage.json

```diff
@@ -4440,7 +4440,7 @@
 												"isExported": true
 											},
 											"comment": {
-												"text": "An array of files to be deleted, including the path and file name. For example [`folder/image.png`].\n"
+												"text": "An array of files to be deletes, including the path and file name. For example [`folder/image.png`].\n"
```

Comment:
`An array of files to be deletes` looks strange to me

Reply:
I think this is pulled from the storage docs, it got updated when i ran the make command

Reply:
yeah, we should update the code comments inside storage-js to solve this